applications/osb-portal/src/components/drawer/WorkspaceDrawer.tsx

```diff
@@ -81,41 +166,101 @@ export const WorkspaceDrawer = (props: any) => {
 
   const handleToggleDrawer = () => props.onToggleDrawer();
 
+  const drawerContent = (open)
+  ? <>
+      <TreeItem
+        nodeId="1"
+        label={
+          <div>Workspace XYZ
+            <SvgIcon className={classes.svgIconRight} viewBox={'0 0 16 16'}>
```

Comment:
What about icons on a variable or svg file to make the rest more readable and to allow reuse?